mo_net/functions.py

```
@@ -71,17 +71,17 @@
     name = ActivationFnName("leaky_relu")
 
     def __call__(self, x: _X) -> _X:
-        if isinstance(x, np.ndarray):
+        if isinstance(x, jnp.ndarray):
             # TODO: fix-types
-            return cast(_X, np.where(x > 0, x, 0.01 * x))
+            return cast(_X, jnp.where(x > 0, x, 0.01 * x))
         else:
             # TODO: fix-types
             return cast(_X, max(0.01 * x, 0))
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)